package.json

```diff
@@ -105,6 +105,7 @@
     "govuk_frontend_toolkit": "^8.2.0",
     "helmet": "^3.21.3",
     "http-proxy-middleware": "^1.0.5",
+    "i": "^0.3.6",
```

Comment:
where do you use this new module 'i'?

Reply:
This must have been added when I was trying to resolve an issue with the rpx-xui-common-lib. I have removed it from the file now.